test/test_confluence.py

```diff
@@ -1,55 +1,70 @@
+import pytest
 from assertpy import assert_that
 
 from label_for_url import determine_label
 
 
-def test_should_label_space() -> None:
-    # Given:
-    url = "https://our-confluence.my-org.de/display/MYSPACE/"
-
+@pytest.mark.parametrize(
+    "url",
+    [
+        "https://our-confluence.my-org.de/display/MYSPACE/",
+        "https://my-org.atlassian.net/wiki/spaces/MYSPACE/overview",
+    ],
+)
+def test_should_label_space(url: str) -> None:
     # When:
     label = determine_label(url)
 
     # Then:
     assert_that(label).is_equal_to("MYSPACE")
 
 
-def test_should_label_url_with_title() -> None:
-    # Given:
-    url = "https://our-confluence.my-org.de/display/MYSPACE/Some+Page+Nobody+Reads"
-
+@pytest.mark.parametrize(
+    "url",
+    [
+        "https://our-confluence.my-org.de/display/MYSPACE/Some+Page+Nobody+Reads",
+        "https://my-org.atlassian.net/wiki/spaces/MYSPACE/pages/1333624914/Some+Page+Nobody+Reads",
+    ],
+)
+def test_should_label_url_with_title(url: str) -> None:
     # When:
     label = determine_label(url)
 
     # Then:
     assert_that(label).is_equal_to("MYSPACE/Some Page Nobody Reads")
 
 
-def test_should_label_url_with_title_and_comment() -> None:
-    # Given:
-    url = (
-        "https://our-confluence.my-org.de/display/MYSPACE/Some+Page+Nobody+Reads"
-        "?focusedCommentId=241754794#comment-241754794"
-    )
-
+@pytest.mark.parametrize(
+    "url",
+    [
+        "https://our-confluence.my-org.de/display/MYSPACE/Some+Page+Nobody+Reads?focusedCommentId=241754794#comment-241754794",
+        "https://my-org.atlassian.net/wiki/spaces/MYSPACE/pages/1333624914/Some+Page+Nobody+Reads?focusedCommentId=241754794",
+    ],
+)
+def test_should_label_url_with_title_and_comment(url: str) -> None:
     # When:
     label = determine_label(url)
 
     # Then:
     assert_that(label).is_equal_to("MYSPACE/Some Page Nobody Reads > Comment 241754794")
 
 
-def test_should_revert_url_encoding() -> None:
-    # Given:
-    url = "https://our-confluence.my-org.de/display/MYSPACE/%5BWIP%5D+Some+Plan+Nobody+Reads"
-
+@pytest.mark.parametrize(
+    "url",
+    [
+        "https://our-confluence.my-org.de/display/MYSPACE/%5BWIP%5D+Some+Page+Nobody+Reads",
+        "https://my-org.atlassian.net/wiki/spaces/MYSPACE/pages/1333624914/%5BWIP%5D+Some+Page+Nobody+Reads",
```

Comment:
Can you confirm that that's how atlassian.net would encode `[WIP]`? Asking because we're seeing umlauts being replaced with `+`.

Reply:
It doesn't, it's just `https://my-org.atlassian.net/wiki/spaces/MYSPACE/pages/1333624914/WIP+Some+Page+Nobody+Reads`